website/index.html

```diff
@@ -698,6 +698,17 @@ <h2 id="keymaps"><a href="#keymaps">Keymaps</a></h2>
 
 };</code></pre>
 
+        <p>Default keymaps: </p>
+
+        <table class="keymaps">
```

Comment:
Why table has this `keymaps` class? 🤔

Reply:
You're welcome.

I was following the same pattern of the other tables as api and config with that styles and etc, but at the end I decided just put the links, so now actually there's no reason to. I can update the commit with no class if you want to.

Reply:
yes please

Reply:
done.